src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Missing header comment. Header comment can be used to describe the purpose of this interface, i.e. what does it represent? Recall from tutorial that an interface is a behavioural contract.

Perhaps `Represents a printable entity.....`